openedx/core/djangoapps/courseware_api/tests/test_views.py

```diff
@@ -156,3 +156,54 @@ def test_resume_with_completion(self):
         assert response.data['block_id'] == str(self.unit.location)
         assert response.data['unit_id'] == str(self.unit.location)
         assert response.data['section_id'] == str(self.sequence.location)
+
+
+@ddt.ddt
+class CelebrationApiTestViews(BaseCoursewareTests):
+    """
+    Tests for the resume API
+    """
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.url = '/api/courseware/celebration/{}'.format(cls.course.id)
+
+    def setUp(self):
+        super().setUp()
+        self.enrollment = CourseEnrollment.enroll(self.user, self.course.id, 'verified')
+
+    @ddt.data(True, False)
+    def test_happy_path(self, update):
+        if update:
+            CourseEnrollmentCelebrationFactory(enrollment=self.enrollment, celebrate_first_section=False)
+
+        response = self.client.post(self.url, {'first_section': True}, content_type='application/json')
+        assert response.status_code == 200
```

Comment:
POST either way? And even on create, returns a 200 instead of a 201? Not big deals, just commenting.

Reply:
Oooh, fancy. OK, fixing.

Reply:
Though yeah, I figured post either way. Just for simplicity. To handle PUT, I'd have to start handing out resource ids for the celebration model and that complexity didn't seem worth it.